### PR TITLE
.NET 8 PR [Merge on 11/14]

### PR DESCRIPTION
**Pull request by jamesmontemagno**

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
+    <AspNetVersion>8.0.0-rc.2.23480.2</AspNetVersion>
```
Comment (bbrandt) [inline Directory.Packages.props:5]: These version numbers can now be `8.0.0` now, right?

Comment (jamesmontemagno) [reply Directory.Packages.props:5]: Yup yup!

File: Directory.Packages.props [hunk 2]
```diff
@@ -18,39 +22,39 @@
...
+    <PackageVersion Include="System.IdentityModel.Tokens.Jwt" Version="7.0.3" />
     <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.5.0" />
     <PackageVersion Include="Swashbuckle.AspNetCore.SwaggerUI" Version="6.5.0" />
     <PackageVersion Include="Swashbuckle.AspNetCore.Annotations" Version="6.5.0" />
     <!-- Test -->
-    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="7.0.8" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="$(AspNetVersion)" />
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.6.3" />
     <PackageVersion Include="xunit" Version="2.4.2" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.5">
```
Comment (WeihanLi) [inline Directory.Packages.props:60]: Maybe we could have these test reference updated also

Comment (jamesmontemagno) [reply Directory.Packages.props:60]: I will create another issue for this.... just to verify any changes

**Comment by jamesmontemagno** [comment]
@WeihanLi  fixed up ;) good catch

**Pull request merged**